README.md

```diff
@@ -72,10 +72,37 @@ can be found [here](lib)
 
 	
 ## Documentation
+
 Detailed documentation can be found [here](docs).
-Usage examples can be found [here](samples)
-	
-## Contributing 
 
-Contributions and suggestions to this product will be possible, and most welcome, 
-starting with next release
\ No newline at end of file
+## Samples
+
+Usage examples can be found [here](samples/src/main/scala)
+
+# Looking for SDKs for other languages/platforms?
```

Comment:
Since spark-connector is just a connector and not a Kusto client library, I'm not quite sure about this...
Maybe we can put links to the related projects (Java client and Python plugin), and explain how they are related? Or just clarify that this is a list of all available Kusto client libraries that can be used in parallel to the connector?